.flake8

```diff
@@ -14,7 +14,7 @@ exclude =
     scripts
     doc
     build_scripts
-    src/command_modules/azure-cli-storage
+    src/command_modules/azure-cli-lab
```

Comment:
This looks like a bad rebase. Lab should be included in flake 8 (and thus shouldn't be in this list).